conference/DB/ecir.yml

```diff
@@ -0,0 +1,18 @@
+  - title: ECIR
+    description: European Conference on Information Retrieval
+    sub: DB
+    rank:
+      ccf: C
+      core: A
+      thcpl: B
+    dblp: ecir
+    confs:
+      - year: 2025
+        id: ecir25
+        link: https://ecir2025.eu/key-dates/
+        timeline:
+          - abstract_deadline: '2024-10-02 23:59:59'
+            deadline: '2024-10-09 23:59:59'
+        timezone: UTC-12
+        date: April 06 – April 10, 2025
```

Comment:
Hi , it seems the dash used `–`  is fall out of ASCII range. pls use this `-`

Reply:
> Hi , it seems the dash `–` falls out of the ASCII range. pls use this `-` instead.

Thank you